src/main/resources/reference.conf

```diff
@@ -56,6 +63,67 @@ jibri {
     audio-source = "alsa"
     // The audio device that will be used to capture audio on Linux
     audio-device = "plug:bsnoop"
+
+    // The path to the ffmpeg binary.
+    ffmpeg-binary = "ffmpeg"
+    ffmpeg = [ ${jibri.ffmpeg.ffmpeg-binary}, "-y", "-v", "info" ]
+    // Input parameters specific to linux
+    input-linux = [
+      "-f", "x11grab"
+      "-draw_mouse", "0"
+      "-r", ${jibri.ffmpeg.framerate}
+      "-s", ${jibri.ffmpeg.resolution}
+      "-thread_queue_size", ${jibri.ffmpeg.queue-size}
+      "-i", ":0.0+0,0"
+      "-f", ${jibri.ffmpeg.audio-source}
+      "-thread_queue_size", ${jibri.ffmpeg.queue-size}
+      "-i", ${jibri.ffmpeg.audio-device}
+    ]
+    // Input parameters specific to macOS
+    input-mac = [
+      "-thread_queue_size", ${jibri.ffmpeg.queue-size}
+      "-f", "avfoundation"
+      "-framerate", ${jibri.ffmpeg.framerate}
+      "-video_size" ${jibri.ffmpeg.resolution}
+      // Note the values passed here will need to be changed based on the output of
+      // ffmpeg -f avfoundation -list_devices true -i ""
+      "-i", "0:0"
+      "-vsync", "2"
+    ]
+    // Audio encoding parameters
+    audio-params = [ "-acodec", "libopus", "-b:a", "64k" ]
+    // Video encoding parameters
+    video-params = [
+      "-c:v", "libx264"
+      "-preset", ${jibri.ffmpeg.video-encode-preset}
+      "-pix_fmt", "yuv420p",
+      "-r", ${jibri.ffmpeg.framerate}
+      "-crf", ${jibri.ffmpeg.h264-constant-rate-factor}
+    ]
+    // Video encoding parameters specific to file recording
+    video-params-recording = ${jibri.ffmpeg.video-params} [
+      "-profile:v", "main"
+      "-level", "3.1"
```

Comment:
Why do you specify a profile and level for recording but not for streaming?

Reply:
No idea, I just ported the existing flags.

Reply:
https://github.com/jitsi/jibri/blob/0271286b4c9dfbe5807a835bd1244e919be47f39/src/main/kotlin/org/jitsi/jibri/sink/impl/StreamSink.kt#L28
https://github.com/jitsi/jibri/blob/0271286b4c9dfbe5807a835bd1244e919be47f39/src/main/kotlin/org/jitsi/jibri/sink/impl/FileSink.kt#L41

Reply:
Ok; while changing/rationalizing these is probably a good idea, we should keep it as a separate PR to avoid being confusing.